src/e2e-test/java/io/cdap/cdap/ui/utils/Constants.java

```diff
@@ -67,4 +69,7 @@ private static double getRandomArbitrary(int min, int max) {
 
   public static final String DEFAULT_GCP_PROJECTID = System.getenv("GCP_PROJECTID");
   public static final String DEFAULT_GCP_SERVICEACCOUNT_PATH = System.getenv("GCP_SERVICE_ACCOUNT_PATH");
+
+  public static final String SINK_PATH_VAL = "/tmp/cdap-ui-integration-fixtures";
```

Comment:
duplicate as `PIPELINE_FILES_DIR` at L41

Reply:
True, but they're used in different ways and I don't want changing one to break the other.